invenio_rdm_records/ext.py

```diff
@@ -88,10 +87,6 @@ def __init__(self, app=None):
     def init_app(self, app):
         """Flask application initialization."""
         self.init_config(app)
-        self.metadata_extensions = MetadataExtensions(
```

Comment:
Config is always loaded [before](https://github.com/inveniosoftware/invenio-base/blob/master/invenio_base/app.py#L105) loading extensions, so if we want to do some validation check, the `ext/init_app` is probably a good place.